src/Illuminate/Redis/Connections/PredisClusterConnection.php

```diff
@@ -2,7 +2,19 @@
 
 namespace Illuminate\Redis\Connections;
 
+use Predis\Command\ServerFlushDatabase;
+
 class PredisClusterConnection extends PredisConnection
 {
-    //
+    /**
+     * Flush the selected Redis database.
+     *
+     * @return void
+     */
+    public function flushdb()
+    {
+        foreach ($this->getConnection()->getIterator() as $node) {
+            $node->executeCommand(new ServerFlushDatabase);
```

Comment:
We are bypassing the `Illuminate\Redis\Connections\Connection::command()` here, which always has been the case, but I wonder if we can fix that in this PR as well.

Reply:
@driesvints: I'm tempted to suggest this to capture the `CommandExecuted` events. Any thoughts?

```php
foreach ($this->client->getIterator() as $node) {
    tap(new PhpRedisConnection($node))->
        ->setEventDispatcher($this->getEventDispatcher())
        ->command('flushdb');
}
```

Reply:
I'll open a separate PR for this.